codex-ui/src/vue/tabbar/Tab.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div :class="[$style.tab]">
+    <div v-if="src">
+      <img
+        :src="src"
+        :class="[$style['tab__left__image']]"
```

Comment:
wrong classname. `__` is used to split elements, there can only be one level of nesting within a block. You need to get rid of second `__` here and further

Sooo, I suggest to rename to 'tab__image'